frontend/app_flowy/test/unit_test/settings_appearance_view_test.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:app_flowy/workspace/presentation/settings/widgets/settings_appearance_view.dart';
+import 'package:flowy_infra/theme.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  late ThemeTypeSetting themeTypeSetting;
+  setUpAll(() {
+    themeTypeSetting = const ThemeTypeSetting(
+      currentThemeType: BuiltInTheme.light,
+    );
+  });
+
+  test(
+    "check theme labels",
+    () {
+      String defaultThemeDisplayName =
+          themeTypeSetting.getThemeNameForDisplaying(BuiltInTheme.light);
+      expect(defaultThemeDisplayName,
+          'settings.appearance.themeType.defaultTheme');
+      defaultThemeDisplayName =
+          themeTypeSetting.getThemeNameForDisplaying(BuiltInTheme.dandelion);
+      expect(defaultThemeDisplayName,
+          'settings.appearance.themeType.dandelionCommunity');
+    },
+  );
+}
```

Comment:
Testing translation keys is kind of unnecessary imo?

Reply:
Yeah, I agree, actually I have never done bloc testing, so didn't do it.